### PR TITLE
APIMF-3091: made embedded json-ld emitter private and hid render option from client

### PR DESCRIPTION
**Pull request by tomsfernandez**

### CODE REVIEW

File: shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ case class RenderOptions(
     filterFields: Field => Boolean = (_: Field) => false,
     amfJsonLdSerialization: Boolean = true,
     useJsonLdEmitter: Boolean = false,
-    flattenedJsonLd: Boolean = false,
+    private[amf] val flattenedJsonLd: Boolean = false,
```
Comment (nschejtman) [inline shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala]: Are we always calling the flattened jsonld emitter directly?

Comment (nschejtman) [reply shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala]: If that is so, we can delete this render option directly. If not shouldn't it be `true` by default?

Comment (tomsfernandez) [reply shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala]: I left it there to use it from the testing module. Yup, should be true by default.

Comment (tomsfernandez) [reply shared/src/main/scala/amf/core/client/scala/config/RenderOptions.scala]: Fixed

**Review by nschejtman — APPROVED** [review]

**Pull request merged**